docs/pages/installation/minikube.md

```diff
@@ -73,17 +73,27 @@ Meshery should now be running in your Minikube cluster and Meshery UI should be
 
 ## Installation: Using `mesheryctl`
 
-Use Meshery's CLI to streamline your connection to your Minikube cluster. Configure Meshery to connect to your Minikube cluster by executing:
+Ensure you are logged in and authenticated by running the following command:
 
-{% capture code_content %}$ mesheryctl system config minikube{% endcapture %}
```

Comment:
Oh, interesting. The `system config minikube` command requires an active token, huh?

Reply:
Hi @jamieplu, yes it does

Reply:
@Alero-Awani why?

Reply:
My apologies, The command requires authentication from either an active token or an existing logged-in session. The token is optional.

Reply:
Thanks, no sweat. Why do we require users to have an active session with Meshery Server in order to execute this command?

Reply:
@leecalcote, I tried to confirm with Uzair before he left. But from what I have seen. This command makes a request to the Meshery Server and retrieves the existing contexts and then uploads the current context to Meshery. So it needs to authenticate with the Meshery Server first. In a situation where the user doesn't run `mesheryctl system login` they get a response that shows they need to login first.

Reply:
@Alero-Awani a chicken-before-the-egg problem. // @lekaf974